### PR TITLE
[SPE-308] show finality times for various evm chains

### PR DESCRIPTION
**Pull request by grikomsn**
## Description

This PR updates the finality time display in tx dialog with mappings from the [Axelar documentation](https://docs.axelar.dev/learn/txduration#common-finality-time-for-interchain-transactions).

## Screenshots

| Avalanche | Ethereum | Polygon |
|-|-|-|
|CleanShot 2023-11-07 at 03 00 32@2x|CleanShot 2023-11-07 at 02 59 46@2x|CleanShot 2023-11-07 at 02 59 31@2x|

### CODE REVIEW

File: src/constants/finality.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  [`${polygon.id}`]: "4:42 minutes",
+  [`${opBNB.id}`]: "46 seconds",
+  [`${fantom.id}`]: "3 seconds",
+  [`${kava.id}`]: "45 seconds",
+  [`${optimism.id}`]: "30 minutes",
+  [`${linea.id}`]: "81 minutes",
+  [`${filecoin.id}`]: "52 minutes",
+  [`${moonbeam.id}`]: "25 seconds",
+  [`${celo.id}`]: "12 seconds",
+  [`${arbitrum.id}`]: "19:06 minutes",
```
Comment (thal0x) [inline src/constants/finality.ts:31]: I think maybe we should just round this up to 20 minutes

File: src/constants/finality.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  polygon,
+} from "wagmi/chains";
+
+import { kava } from "./constants";
+
+/** @see https://docs.axelar.dev/learn/txduration#common-finality-time-for-interchain-transactions */
+const finalityTimeMap: Record<string, string> = {
+  [`${ethereum.id}`]: "16 minutes",
+  [`${avalanche.id}`]: "3 seconds",
+  [`${polygon.id}`]: "4:42 minutes",
```
Comment (thal0x) [inline src/constants/finality.ts:22]: I think maybe we should just round this up to 5 minutes

**Review by thal0x — APPROVED** [review]
Small comments but otherwise lgtm

File: src/constants/finality.ts (hunk 2 above)
Comment (grikomsn) [reply src/constants/finality.ts:22]: @thal0x like this?

```suggestion
  [`${polygon.id}`]: "~5 minutes",
```

File: src/constants/finality.ts (hunk 1 above)
Comment (grikomsn) [reply src/constants/finality.ts:31]: @thal0x also like this?

```suggestion
  [`${arbitrum.id}`]: "~20 minutes",
```

Comment (thal0x) [reply src/constants/finality.ts:31]: yeah perfect

File: src/constants/finality.ts (hunk 2 above)
Comment (thal0x) [reply src/constants/finality.ts:22]: yep!

**Review by thal0x — APPROVED** [review]

**Pull request merged**